src/macos/Animated.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+* Animated.tsx
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* RN-specific implementation of the cross-platform Animation abstraction.
+*/
+import React = require('react');
```

Comment:
I don't like the fact that we're duplicating this code for every platform. We should look for a better way to share this code. If you have an idea for how to do this, feel free. Otherwise, it can wait until some future change.

Reply:
Agree, I think constant ReactNativeAnimatedClasses is the only part that needs to be modified for platform specific needs, Animated wrappers look the same to me (I will need to check windows implementation first).